index.html

```diff
@@ -0,0 +1,157 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+      
+    <link rel="stylesheet" href="./fontawesome/css/all.css" />
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <title>Sohaib Capstone</title>
+</head>
+<body>
+    <header>
+        <nav class="black-navbar">
+            <a href="#">
+                <i class="fa-brands fa-facebook-f"></i>
+            </a>
+            <a href="#">
+                <i class="fa-brands fa-twitter"></i>
+            </a>
+            <a href="#">Logout</a>
+            <a href="#">MyPage</a>
+            <a href="#" class="last">Dubai</a>
+        </nav>
+
+        <nav class="white-navbar">
+            <div class="desk-logo">
+                <a href="index.html">
+                    <img src="./images/index logo.png" alt="Index Page Logo">
+                </a>
+            </div>
+            <ul class="nav-itmes">
+              <li>
+                <a class="nav-links first" href="about.html">About</a>
+              </li>
+              <li>
+                <a class="nav-links" href="#program">Program</a>
+              </li>
+              <li>
+                <a class="nav-links" href="#">Join</a>
+              </li>
+              <li>
+                <a class="nav-links" href="#sponser">Sponser</a>
+              </li>
+              <li>
+                <a class="nav-links" href="#">News</a>
+              </li>
+              <li>
+                <a href="#">
+                    <button class="nav-btn">Campaign</button>
+                </a>
+              </li>
+            </ul>
+        </nav>
+        <nav class="mobile-nav">
+            <div class="close-btn">
+                <i class="fa-solid fa-xmark"></i>
+            </div>
+          <ul class="nav-list">
+            <li>
+              <a class="nav-links" href="about.html">About</a>
+            </li>
+            <li>
+              <a class="nav-links" href="#program">Program</a>
+            </li>
+            <li>
+              <a class="nav-links" href="#">Join</a>
+            </li>
+            <li>
+              <a class="nav-links" href="#">Sponser</a>
+            </li>
+            <li>
+              <a class="nav-links" href="#">News</a>
+            </li>
+            <li>
+              <a href="#">
+                  <button class="mobile-nav-btn">Campaign</button>
+              </a>
+            </li>
+          </ul>
+        </nav>  
+        <div class="menu-btn">
+            <i class="fa-sharp fa-solid fa-bars"></i>
+        </div >
+     <div class="header-info">
+        <div class="summit-header">
+            <img src="./images/summit.png" alt="First Header">
+        </div>
```

Comment:
- [ ] Please do not use images to simulate the headings of your page. This should be the main heading and you should use an `<h1>` tag to wrap it. Please create this heading for this page and try to style as much as possible as this heading you copied.

Reply:
Done